### DISCUSSION TITLE
Support for newer SiLabs EFR32 parts

### DISCUSSION THREAD

**Discussion by brian-level** [openthread/openthread#8346] [Q&A] [answered]
I am hoping to evaluate OpenThread on SiLabs EFR32 parts.  The "Series 1" parts that are supported in ot-efr32 aren't currently available in quantity, especially the EFR32MG12.  To use the newer Series-2 parts  I think what is needed are RAIL and NVM libraries with the proper configurations for OpenThread.  I can build the rest of the stack from ot-efr32 but don't know how to get/create the RAIL library. Also, I'd really like to look at Sub-GHz thread on series-2 parts like EFR32FG23.  Is there someone with Sub-GHz OpenThread experience here that can share some experiences?

**Comment by silabs-YupingX** [comment]
@brian-level Since sub-GHz OpenThread is not part of the current OpenThread spec, we don’t have a standard solution. However, we do have single band Proprietary Sub-GHz support, please refer to [AN1350](https://www.silabs.com/documents/public/application-notes/an1350-openthread-single-band-proprietary-sub-ghz.pdf). If this is something you might be interested in and if you have more questions, we can have an offline discussion.

**Comment by brian-level** [comment]
That AN1350 was exactly what got me on this path to Sub-GHz.

We don't actually need to be Thread "certified" or compliant.  We just need
a mesh network that can operate over a large (building scale) area and
coexist with Zigbee and WiFi.   In my testing I've found 900 works a lot
better than 2.4 in this scenario.  When looking in to developing a mesh
network on 900, I looked at things like WiSun, etc but thought anything I
did would be basically like Thread in general, so why not start with
something that has a lot of bugs taken out already.

At this point my big question is "How do I get this on a chip I can buy
today".

Please lets talk more about this.

Brian

**Comment by brian-level** [comment, marked as answer]
For anyone that finds this, it is indeed possible.  Start with configuring a RAIL for your chip based on information in openthread and AN1350.  The rest is a lot of convincing the SiLabs tools your chip and build exist and debugging
